src/utils/parsers.rs

```diff
@@ -20,6 +22,62 @@ pub fn parse_schema_source(loc: &str) -> Result<SchemaSource> {
     }
 }
 
+#[derive(Debug, Clone)]
+pub struct GraphRef {
+    pub name: String,
+    pub variant: String,
+}
+
+/// NOTE: THIS IS A TEMPORARY SOLUTION. IN THE FUTURE, ALL GRAPH ID PARSING
+/// WILL HAPPEN IN THE BACKEND TO KEEP EVERYTHING CONSISTENT. THIS IS AN
+/// INCOMPLETE PLACEHOLDER, AND MAY NOT COVER EVERY SINGLE VALID USE CASE
```

Comment:
We've decided to move forward with a very TEMPORARY graph ref parser to parse graph@variant positional args. This will be stripped out as soon as the API supports a graph ref, allowing all parsing and validation of these refs on the backend